src/sage/combinat/sf/dual_grothendieck.py

```diff
@@ -0,0 +1,82 @@
+"""
+Dual Grothendieck Symmetric Functions
+"""
+
+from sage.categories.hopf_algebras import HopfAlgebras
+from sage.combinat.free_module import CombinatorialFreeModule
+from sage.combinat.partition import Partition, Partitions
+from sage.combinat.sf.sf import SymmetricFunctions
+from sage.combinat.skew_tableau import SemistandardSkewTableaux
+from sage.rings.polynomial.polynomial_ring_constructor import PolynomialRing
+
+
+class DualGrothendiecks(CombinatorialFreeModule):
+    """
+    Dual (symmetric) Grothendieck functions.
+
+    EXAMPLES::
```

Comment:
```suggestion
class DualGrothendiecks(CombinatorialFreeModule):
    """
    Dual (symmetric) Grothendieck functions.
    
    Dual Grothendiecks come from the K-theory of the Grassmannian.

    EXAMPLES::
```
Here is how to suggest a change.